### PR TITLE
Pyroute2

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: conf/build.manifest [hunk 1]
```diff
@@ -294,6 +299,8 @@
 				"pkg-pybonjour",
 				"pydevd",
 				"python-netsnmpagent",
+				"python-pyroute2",
+				"python-remote-pdb",
```
Comment (william-gr) [inline conf/build.manifest:303]: Do we need these two here? isnt in middlewared repo enough?

Comment (themylogin) [reply conf/build.manifest:303]: Looks like we do, this package wasn't built during the last build

Comment (william-gr) [reply conf/build.manifest:303]: So yes for python-pyroute2, but no for python-remote-pdb ?

Comment (themylogin) [reply conf/build.manifest:303]: How is it different? The same thing will happen if we upgrade it, it will only be rebuild during full build.

Comment (william-gr) [reply conf/build.manifest:303]: I missed the remote-pdb being a repo of our own

**Review by william-gr — APPROVED** [review]

**Pull request merged**